src/esphome/sensor/mpr121_sensor.h

```diff
@@ -0,0 +1,63 @@
+#ifndef ESPHOME_SENSOR_MPR121_H
+#define ESPHOME_SENSOR_MPR121_H
+
+#include "esphome/defines.h"
+
+#ifdef USE_MPR121
+
+#include "esphome/sensor/sensor.h"
+
+ESPHOME_NAMESPACE_BEGIN
+
+namespace sensor {
+
+enum {
+  MPR121_SENSOR_TYPE_SLIDER = 0x00,
+  MPR121_SENSOR_TYPE_ROTARY = 0x01,
+}; 
+
+/** SensorChannel class just a placeholder for each mpr121 sensor channel.
+ *
+ * The channels together form one Sensor. 
+ * The Sensor value is calculated by the values of the channels that are active.
+ */
+class MPR121SensorChannel {
+ public:
+  MPR121SensorChannel(uint8_t channel_num = 0, float value = 0.0);
+
+  uint8_t channel = 0;
```

Comment:
`protected` and trailing underscore.

Reply:
channel is accessed from void MPR121Sensor::process_slider_(uint16_t data).
I can make channel protected, but then I need a getter to access it.